Sources/StreamVideo/OpenApi/generated/Models/BlockUserRequest.swift

```diff
@@ -1,30 +1,18 @@
 //
-// BlockUserRequest.swift
-//
-// Generated by openapi-generator
-// https://openapi-generator.tech
+// Copyright © 2024 Stream.io Inc. All rights reserved.
 //
 
 import Foundation
-
-
-public struct BlockUserRequest: Codable, JSONEncodable, Hashable {
-    /** the user to block */
+    
+public struct BlockUserRequest: @unchecked Sendable, Codable, JSONEncodable, Hashable {
```

Comment:
All the other ones moderation related - we should probably have a separate package.

Reply:
This one is used in `/video/call/{type}/{id}/block`. Are you sure we don't want this?